run_e3sm.template.sh

```diff
@@ -24,8 +24,10 @@ readonly PROJECT="e3sm"
 # Simulation
 readonly COMPSET="WCYCL1850"
 readonly RESOLUTION="ne30pg2_EC30to60E2r2"
-readonly CASE_NAME="v2.LR.piControl"
-readonly CASE_GROUP="v2.LR"
+# TODO: CHANGE the following CASE_NAME to desired values
```

Comment:
Is this an actual TODO for the developers/maintainers of `run_e3sm.template.sh` or is it a direction for people who copy this template? If the latter, something like ` # IMPORTANT` or `# BEFORE RUNNING` might be better.

Reply:
It's for the user to do. I can change to "BEFORE RUNNING"